pkg/announcements/types.go

```diff
@@ -58,6 +58,17 @@ const (
 
 	// ---
 
+	// ServiceAccountAdded is the type of announcement emitted when we observe an addition of a Kubernetes Service Account
+	ServiceAccountAdded AnnouncementType = "serviceaccount-added"
+
+	// ServiceAccountDeleted the type of announcement emitted when we observe the deletion of a Kubernetes Service Account
+	ServiceAccountDeleted AnnouncementType = "serviceaccount-deleted"
+
+	// ServiceAccountUpdated is the type of announcement emitted when we observe an update to a Kubernetes Service
+	ServiceAccountUpdated AnnouncementType = "serviceaccount-updated"
```

Comment:
I completely understand why we're creating a dependency on watching service account changes to generate the rules for inbound routes but I think we should revisit if we can get rid of this while we're implementing RBAC filters on routes.

Reply:
I agree, let me add a note on the RBAC filters issue to revisit and eliminate this if possible

Reply:
Thank you!